flink-connectors/flink-connector-kafka/src/main/java/org/apache/flink/streaming/connectors/kafka/table/KafkaConnectorOptionsUtil.java

```diff
@@ -241,6 +291,26 @@ public static StartupOptions getStartupOptions(ReadableConfig tableOptions) {
         return options;
     }
 
+    public static EndOptions getEndOptions(ReadableConfig tableOptions) {
+        final Map<KafkaTopicPartition, Long> specificOffsets = new HashMap<>();
+        final EndMode endMode =
+                tableOptions
+                        .getOptional(SCAN_END_MODE)
+                        .map(KafkaConnectorOptionsUtil::fromOption)
+                        .orElse(EndMode.END_GROUP_OFFSETS);
```

Comment:
The default value for the bounded mode should be unbounded. If the source is unbounded, there will not be specific offsets for the source.